scripts/bindings.py

```diff
@@ -8,26 +8,27 @@
 
 """
 
-
 def return_lines(filename, start, end):
     """
     Returns only the lines of a file between start and end.
 
     :param filename: the filename
-    :param start: first line
-    :param end: last line
-    :return: list of lines
+    :param start: text of first line to match
+    :param end: text of last line match
+    :return: generator of lines between and including the matching lines
     """
 
-    lines = []
-
-    with open(filename, 'r') as fin:
+    accumulate = False
 
-        for lineno, line in enumerate(fin):
-            if start <= lineno + 1 <= end:
-                lines.append(line.strip())
+    with open(filename, 'rt', encoding='utf-8') as fin:
 
-    return lines
+        for line in fin:
+            if not accumulate and start in line:
+                accumulate = True
+            if accumulate:
+                yield line.strip()
+            if accumulate and end in line:
+                break
```

Comment:
This complicates what is just an utility. I've implemented this to just run once or twice "per year", i.e., just to create the initial function signatures. There's no need to do complex things with this such as returning generators, even more that this returns few lines of texts.